.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,5 +25,18 @@ labels: bug
 
 ### Expected behaviour
 
+### Minimal init.vim or init.lua and code sample
+<!-- You can download a minimal_init.lua from here 
+      curl -fLO https://raw.githubusercontent.com/neovim/nvim-lspconfig/master/test/minimal_init.lua 
+      After editing to include your language server, run neovim with nvim -u minimal_init.lua -->
+
 ### LSP log
-<!-- Please add vim.lsp.set_log_level("debug") to your lua block in init.vim and paste a link to your log file, located at $HOME/.local/share/nvim/lsp.log -->
+<!-- If not using the minimal_init.lua please add vim.lsp.set_log_level("debug") to your lua block 
+     in init.vim and paste a link to your log file, located at $HOME/.local/share/nvim/lsp.log -->
```

Comment:
unrelated but we should put this log in .cache rather than XDG_DATA_HOME

Reply:
Filed an issue in core, I can do it